avaframe/ana5Utils/distanceTimeAnalysisCfg.ini

```diff
@@ -44,9 +44,41 @@ interpMethod = bilinear
 # dsMin meters after the beta point also have an angle bellow 10°
 dsMin = 30
 
+# use the maximum rangeTimeResType or the mean rangeTimeResType (max or mean) values computed for crossprofiles along path
+maxOrMean = mean
+
+# use flow parallel or projected distance along flow path for plots and to compute approach velocity
+# options: projected, parallel
+sType = projected
+
 
 [PLOTS]
 # every gateContours gate range plotted as contour line in radar field of view plot
 gateContours = 20
 # show debug plots
 debugPlot = False
+# width of velocity legend in fraction of full figure width
+width = 0.25
+# height of velocity legend in fraction of full figure height
+height = 0.25
+# line width of velocity legend lines
+lw = 0.25
+# text size of velocity legend
+textsize = 7
+# if an animation of flow result, transformed into TT domain and TT-diagram shall be created
+# if True uncomment section ANIMATE and add values to parameters
+animate = False
+
+# [ANIMATE]
+# set extent of tt-diagram and result type min and max
+# uncomment the following parameters and set a value - required if animate True in section PLOTS
+# resMin =
+# resMax =
+# xMin =
+# xMax =
+# yMax =
+# yMin =
+# annotate x, y with East and North in plot
+# xyEastNorth = True
+# add titles to panels
+# panelTitles = True
```

Comment:
I agree! I can also uncomment them section to avoid this issue but leave the parameters empty so you get an error and reminded to first put some values, because putting some for the parabola only makes sense for the parabola then..

Reply:
but I added the parameters for the parabola